sirepo/template/activait.py

```diff
@@ -175,6 +175,23 @@ def get_fft_report(run_dir, data):
     return w, plots, f"FFT", summaryData
 
 
+def new_simulation(data, new_sim_data, qcall, **kwargs):
+    if "sourceSimType" not in new_sim_data:
+        return
+    t_basename = f"{new_sim_data.sourceSimType}-{new_sim_data.sourceSimId}-{new_sim_data.sourceSimFile}"
+    data.models.dataFile.dataOrigin = "file"
+    data.models.dataFile.file = t_basename
+    t = simulation_db.simulation_lib_dir(_SIM_DATA.sim_type(), qcall=qcall).join(
+        _SIM_DATA.lib_file_name_with_model_field("dataFile", "file", t_basename)
+    )
+    if t.exists():
+        return
+    s = simulation_db.simulation_dir(
+        new_sim_data.sourceSimType, sid=new_sim_data.sourceSimId, qcall=qcall
+    ).join(new_sim_data.sourceSimFile)
+    t.mksymlinkto(s, absolute=False)
```

Comment:
You should note that this is hack. `#TODO(mkeilman) Only works on local installation, not sbatch/NERSC` or something like that.

Reply:
Whoops, Paul was too fast.  I'll clean these up along the way.